app/scripts/services/browserStorage.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('BrowserStorage', function() {
+
+    // Automaticaly namespaces all of our storage, wether in session or local
+    var namespace = 'openshift-';
```

Comment:
We've started using `openshift/`, so let's use that here. For instance,

https://github.com/openshift/origin-web-common/blob/master/src/services/recentlyViewedProjects.js#L6